### DISCUSSION TITLE
How does Minio handle data replication behind the scenes?

### DISCUSSION THREAD

**Discussion by milandangol** [minio/minio#17422] [Q&A] [answered]
Hey everyone! I'm new to Minio and looking to replicate data between sites. Can someone explain how Minio achieves data replication? Does it rely on its client, mc, or does it use another method for replication? Thanks!

**Comment by klauspost** [comment, marked as answer]
Everything is explained here: https://min.io/docs/minio/kubernetes/upstream/administration/bucket-replication.html
